src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,126 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int BIT_SHIFT = 1;
+    private static final int LIST_ELEMENTS_FIRST_INDEX = 0;
+
+    private T[] listElements;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 - is the default value for int data type